client/src/main/java/com/influxdb/client/OrganizationsQuery.java

```diff
@@ -0,0 +1,96 @@
+package com.influxdb.client;
+
+import javax.annotation.Nullable;
+
+public class OrganizationsQuery {
+
+    /**
+     * offset  (optional)
```

Comment:
Fix `JavaDoc` formatting:

```suggestion
     * Offset. (optional)
```